tests/test_acasclient.py

```diff
@@ -209,6 +218,7 @@ def setUp(self):
         creds = acasclient.get_default_credentials()
         self.client = acasclient.client(creds)
         self.tempdir = tempfile.mkdtemp()
+        self.maxDiff = None
```

Comment:
maxDiff seems to be unused.

Reply:
Just added a comment, its a unittest thing